app/controllers/feedback/post-index.controller.js

```diff
@@ -14,23 +14,20 @@ ${req.body['feedback-suggestion']}`
     email: req.body.email,
     name: '(no name supplied)',
     type: 'question',
-    subject: `Feedback from service`,
+    subject: `Feedback from: ${req.body['service-name']}`,
```

Comment:
Do we ever have `service-name` form field available as we don't use service/account middleware anymore ?

Reply:
Thanks Krishna - made a mistake when updating the code - this is now `Feedback from service`